sandbox/data/custom-node/custom-node.config.js

```diff
@@ -1,5 +1,5 @@
-import React from 'react';
-import CustomNode from './CustomNode';
+import React from "react";
```

Comment:
Cool glad that prettier ran against this file :clap: . But please please discard the changes to the file ` sandbox/rd3g.sandbox.bundle.js`, that should only be modified on a release commit.